src/store/account/account.store.impl.ts

```diff
@@ -55,14 +55,13 @@ export class AccountStoreImpl extends StoreImpl implements AccountStore {
     this.safient = new SafientCore(
       this.signer,
       Enums.NetworkType.localhost,
-      'threadDB',
+      Enums.DatabaseType.threadDB,
```

Comment:
There's no need to bind enums inside Enums. `DatabaseType.threadDB` would do. Enums declarations must go into respective folders based on their context.